homeassistant/components/mysensors/gateway.py

```diff
@@ -92,6 +94,7 @@ def gateway_ready_callback(msg):
         )
         if gateway is None:
             return False
+        gateway.on_conn_made = on_conn_made
 
         connect_task = None
         try:
```

Comment:
I think we should try to harmonize the try_connect and start_gateway flows, eg use the same timeout constant.